storage-transcribe-audio/functions/src/transcribe-audio.ts

```diff
@@ -0,0 +1,52 @@
+
+import {SpeechClient} from '@google-cloud/speech';
+import {google} from '@google-cloud/speech/build/protos/protos';
+import {ObjectMetadata} from 'firebase-functions/v1/storage';
+import * as logs from './logs';
+
+export async function transcribeAudio(
+  client: SpeechClient, object: ObjectMetadata): Promise<void> {
+
+  const request = {
+    config: {
```

Comment:
Are these user-configurable? Should they be params?

Reply:
Some of this will depend on the sample rate of the specific file; the encoding can always be LINEAR16 because best practices ask us to transcode it (https://cloud.google.com/speech-to-text/docs/best-practices).

The language code should be configurable, though; yeah. Let's talk more about this off the issue

Reply:
Let's make LINEAR16 and 48000 constants near the top of the file.

Reply:
LINEAR16 can be a constant at the top, but 48000 will come from the ffprobe call